### ISSUE TITLE
Tag releases

### ISSUE THREAD

**Issue by lukemelia** [Kerry350/ember-deploy-s3-index#19]
It would be great if this project tagged releases that were published to npm so it is possible to easily see what version corresponds to what point in the git commit history.

**Comment by Kerry350** [comment]
Agreed. They will be tagged from now on, starting from now with `0.4.0`.

**Comment by jrjohnson** [comment]
Any chance of getting a new release?  The silent error deprecation warning is getting to me.
